mkdocs.yml

```diff
@@ -1,94 +1,72 @@
 nav:
     - Home:
         - Introduction: index.md
-        - About: why.md
-        - Glossary: glossary.md
-        - License: license.md
-        - Friends: friends.md
-        - F.A.Q.: faq.md
+        - About: about.md
     - Getting Started Guide:
-        - Downloading an image: getting_image.md
-        - First boot: first_boot.md
-        - Network setup: setup_network.md
-        - Configuration: setup_ovos.md
-    - Features and skills:
-        - Standard skills: standard_skills.md
-        - Playing music: playing_music.md
-        - Playing video: playing_video.md
-    - Smartspeaker functionality:
-        - Airplay: airplay.md
-        - Spotify connect: spotifyd.md
-        - Bluetooth speaker: btspeaker.md
-        - KDE Connect: kdeconnect.md
-    - OVOS-core:
-        - Introduction: core.md
-        - Bus: bus_service.md
-        - Skills: skills_service.md
-        - Speech: speech_service.md
-        - Audio: audio_service.md
-        - GUI: gui_service.md
-        - Configuration: config.md
-    - Skills:
-        - Introduction: skills.md
-        - Installing new skills: skill_install.md
-        - Design Guidelines: guidelines.md
-        - Anatomy: skill_structure.md
-        - Dialog: statements.md
-        - Prompts: prompts.md
-        - Intents: intents.md
-        - Context: context.md
-        - Converse: converse.md
-        - Filesystem: skill_filesystem.md
-        - Skill Settings: skill_settings.md
-        - Skill GUI: skill_gui.md
-        - Skill API: skill_api.md
-        - Skill Json: skill_json.md
-        - Skill Tests: skill_tests.md
-        - F.A.Q.: dev.md
-    - GUI:
-        - Introduction: gui_intro.md
-        - Homescreen: homescreen.md
-        - QML Design Guidelines: qml_guidelines.md
-        - Open Linguistika: linguistika.md
-        - Shell: shell.md
-        - Protocol: gui_protocol.md
-        - Extensions: gui_extensions.md
-        - Bigscreen: bigscreen.md
-    - Backend:
-        - Available Backends: backend.md
-        - Backend Client: backend_client.md
+        - Hardware: gs_hardware.md
+        - Installation Choices: gs_installation_choices.md
+        - Installing an Image: gs_installing_image.md
+        - First boot: gs_first_boot.md
+        - Next Steps: gs_next_steps.md
+    - Architecture:
+        - OpenVoiceOS Architecture: architecture.md
+        - Backend: arch_backend.md
+        - APIs: arch_apis.md
+        - Services: arch_services.md
+        - Plugins: arch_plugins.md
+        - PHAL: arch_phal.md
+        - Skills: arch_skills.md
+        - Statements: statements.md
+    - Hardware:
+        - Platforms: hardware_platforms.md
+        - Audio: hardware_audio.md
+        - Video: hardware_video.md
+    - Installation:
+        - Installation Introduction: install_intro.md
+        - Install OVOS Core: install_ovos_core.md
+        - Installing Skills: install_skills.md
+        - Raspbian: install_raspbian.md
```

Comment:
Maybe move this down to the images section since there is now an image available and I will add how to build manually.